content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+
+Predefined blueprints are provided when the service is provisioned, and you can create your own custom machine blueprints.
+
+A machine blueprint includes the:
+
+* Machine provider
+* Operating system image and version
+* Number of vCPU cores and amount of memory in the node
+* Compute instance types
+* Storage instance types
+
+![](/img/image-1.png)
+
+
+
+![](/img/image-2.png)
+
+
+
+![](/img/image-3.png)
+
+## HPE GreenLake for Containers: Cluster Blueprint layout for Kubernetes cluster
+
+HPE GreenLake for Containers uses cluster blueprints to define the cluster layout and other infrastructure details used to create a cluster.
+
+Predefined blueprints are provided when the service is provisioned. You can copy and modify the predefined blueprints or create your own custom cluster blueprints.
+
+A cluster blueprint includes the:
+
+* Cluster provider
+* Version of Kubernetes to deploy on the cluster
+* Default storage class
+* Control plane nodes and worker nodes
+
+![](/img/image-4.png)
+
+
+
+![](/img/image-5.png)
+
+## HPE GreenLake for Containers: MongoDB Application deployment on Kubernetes cluster
+
+MongoDB is an open source, NoSQL database that provides support for JSON-styled, document-oriented storage systems. It supports a flexible data model that enables you to store data of any structure, and provides a rich set of features, including full index support, sharding, and replication.
+
+Below is the preferred cluster configuration for MongoDB In-Memory Database workload
+
+![](/img/image-6.png)
+
+
+
+![](/img/image-7.png)
+
+### Step-1: Create Kubernetes cluster from Containers page
+
+To create a cluster, you must have the roles Private Cloud Cluster Owner and Private Cloud Widget Viewer. 
+
+* From the Containers main page, under the Clusters tab, click Create Cluster. 
+* In the Create Cluster form, provide the cluster name 'hpe', and select the standard cluster blueprint. The new cluster appears in the list of clusters.
+
+![](/img/image-8.png)
+
+
+
+![](/img/image-9.png)
+
+As indicated above there are multiple clusters deployed in parallel for multiple purposes. For MongoDB Application deployment we will use cluster created with name 'hpe'.
+
+![](/img/image-10.png)
+
+### Step-2: Download scoped kubeconfig from Container platform page
+
+* From the Clusters tab, select the 'hpe' Kubernetes cluster and click Launch Service Console. You are directed to the container platform page.
+* Click on Download kubeconfig.
+
+![](/img/image-11.png)
+
+
+
+![](/img/image-12.png)
+
+Note: Launching HPE Ezmeral Runtime Enterprise from HPE GreenLake Central is configured through SAML SSO and adds a session token to the kubeconfig file. You will need to download the kubeconfig file again if you want to continue to access the cluster when the session token expires after an hour. 
+
+### Step-3: View the 'hpe' Kubernetes cluster environment details
+
+View the Kubernetes version
```

Comment:
Could you add a little more to this to clarify what you mean? View the Kubernetes version of what? Ah, wait... I think I know what you're trying to do. It may help, before you start enumerating the different things you can view, to have a single sentence just below the Step-3 header that says something like "There are a number of things you can view from here. For instance, you can:"

Reply:
Got your point, after the header, in code block i will keep the details. mean while removed the line itself. i.e. View the Kubernetes version.